test/unit/vm/disposable_vm_test.dart

```diff
@@ -0,0 +1,355 @@
+// Copyright 2016 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:async';
+
+import 'package:test/test.dart';
+
+import 'package:w_common/disposable_vm.dart';
+
+import './vm_stubs.dart';
+import '../stubs.dart';
+
+void main() {
```

Comment:
All of the unit tests in this file are duplicated in `disposable_browser_test.dart`. It seems like these tests could be deleted from `disposable_browser_test.dart` and that file could just focus on the methods that are added by that class.